stock_card_report/reports/stock_card_report_xlsx.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+
+class ReportStockCardReportXlsx(models.TransientModel):
```

Comment:
models.AbstractModel ?

Reply:
I changed `models.TransientModel` to `models.AbstractModel`.